xds/src/test/java/io/grpc/xds/internal/sds/SecretVolumeSslContextProviderTest.java

```diff
@@ -274,7 +274,8 @@ public void getProviderForServer_defaultTlsCertificate_throwsException() {
     try {
       SecretVolumeSslContextProvider.getProviderForServer(
           CommonTlsContextTestsUtil.buildDownstreamTlsContext(
-              CommonTlsContextTestsUtil.getCommonTlsContext(tlsCert, /* certContext= */ null)));
+              CommonTlsContextTestsUtil.getCommonTlsContext(tlsCert, /* certContext= */ null),
+              /* requireClientCert= */ false));
```

Comment:
we don't need those changes anymore.

Reply:
this is a different one. This one doesn't take filenames but just a CommonTlsContext and the new bool. I could have created the same 2 methods here too but this is not so widely used

Reply:
oops okay, ideally passing boolean param is not recommended if can avoid. but this one is not that bad.